generate_txt.ps1

```diff
@@ -0,0 +1,40 @@
+$Directory = Resolve-Path $Args[0]
+$LineEnding = 1 # WdLineEndingType 1 = CROnly
+$Encoding = 65001 # Codepage for UTF8
+
+# Read backlist
+# $Blacklist = @()
+# $parent = (get-item $Directory).parent.parent.parent
+# $bl_file = Join-Path -Path $parent -ChildPath "blacklist.txt"
+# foreach ($b in (Get-Content $bl_file)) {
+#     $Blacklist += $b
+# }
+
+$AbsPath = @(Get-ChildItem -path $Directory -Recurse -Exclude *.txt, *.skip)
+:main for($i=0; $i -lt $AbsPath.length; $i++) {
+    $AbsPathI = Resolve-Path $AbsPath[$i] | Select-Object -ExpandProperty Path
+
+    if (Test-Path ($AbsPathI + ".txt") -PathType Leaf ) { continue main }
+
+    # $filename = Split-Path $AbsPath[$i] -leaf
+    # $join_fn = Join-Path -Path $Args[0] -ChildPath $filename
+    <# if (($join_fn | Out-String) -contains $Blacklist) { 
+        Write-Output "Skipping $join_fn"
+        continue main
+    } #>
```

Comment:
Is the blacklist no longer being used?

Reply:
At the moment, it's currently not - but I think the more reasonable solution (in the future) is to make a blacklist, which is why I left it there (for now).